src/patternfly/components/Sidebar/sidebar.scss

```diff
@@ -148,6 +157,20 @@ $pf-c-sidebar__panel--list--width: ("default", 25, 33, 50, 66, 75, 100);
     --pf-c-sidebar__panel--Position: static;
     --pf-c-sidebar__panel--Top: auto;
   }
+
+  > .pf-c-tabs {
+    --pf-c-tabs--m-vertical--MaxWidth: auto; // when tabs are part of the sidebar, unset max width
```

Comment:
This was also set in the tabs component. Did you mean to just make these modifications here locally in the sidebar component? I know some people are using tabs in the sidebar now, even though the sidebar is beta. I wonder if it makes sense for this to be a variation? And/or the sidebar panel has width modifiers - maybe we should add an `auto` width modifier for things like this with a predefined width? Seems like adding both variations would make it the most flexible and be useful. Ultimately this should just be the default though, like jump links.